Remove `snapshot_ctx_repository` attr from pipeline definition

8R0WNI3 opened the pull request:
This attribute will be removed with a future release of cc-utils. Hence, it must be removed from the pipeline definition as well. However, this will have no impact on the current functionality as the  will be used as default for both snapshot
and non-snapshot component descriptors.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:
<!--  Write your release note:
1. Enter your release note in the below block.
2. If no release note is required, just write "NONE" within the block.

Format of block header: <category> <target_group>
Possible values:
- category:       breaking|feature|bugfix|doc|other
- target_group:   user|operator|developer|dependency
-->
```feature user

```

gardener-robot-ci-3 commented:
Thank you @8R0WNI3 for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

8R0WNI3 committed:
Remove `snapshot_ctx_repository` attr from pipeline definition

This attribute will be removed with a future release of cc-utils.
Hence, it must be removed from the pipeline definition as well.
However, this will have no impact on the current functionality as
the `ocm_repository` will be used as default for both "snapshot"
and "non-snapshot" component descriptors.
